### PR TITLE
feat: update Angular packages to version 13.0.0

### LINKED ISSUE #3189: Build: Update to latest Angular v13 RC
## If accepted, I would be willing to submit a PR for this feature

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

### PR DESCRIPTION
**Pull request by brandonroberts**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #3189

## What is the new behavior?

The minimum version of Angular has changed

BEFORE:

The minimum version of Angular is 12.x

AFTER:

The minimum version of Angular is 13.x

## Does this PR introduce a breaking change?

```
[x] Yes
[ ] No
```

## Other information

~~Blocked on https://github.com/thymikee/jest-preset-angular/issues/1090~~

### CODE REVIEW

File: modules/component/schematics-core/tsconfig.lib.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "downlevelIteration": true,
-    "outDir": "../../dist/modules/schematics-score",
```
Comment (timdeschryver) [inline modules/component/schematics-core/tsconfig.lib.json:8]: I actually like the nam schematics-core 👀

File: package.json [hunk 2]
```diff
@@ -186,31 +186,35 @@
     "karma-jasmine-html-reporter": "1.5.4",
     "lint-staged": "^8.0.0",
     "ncp": "^2.0.0",
-    "ng-packagr": "12.2.1",
+    "ng-packagr": "13.0.3",
     "npm-run-all": "^4.1.5",
     "nyc": "^10.1.2",
     "ora": "^1.3.0",
+    "postcss": "^8.3.9",
```
Comment (timdeschryver) [inline package.json:193]: Out of interest, why where these imports added?

File: package.json [hunk 3]
```diff
@@ -31,6 +31,7 @@
     "bazel:lint": "yarn bazel:format --lint=warn",
     "bazel:lint-fix": "yarn bazel:format --lint=fix",
     "copy:schematics": "ts-node ./build/copy-schematics-core.ts",
+    "schematics:check": "git diff --name-only --exit-code ./modules",
```
Comment (timdeschryver) [inline package.json:34]: Was the intention to use this script in the CI?
If it was, then I think we forgot to update the config.yml 😅

**Review by timdeschryver — APPROVED** [review]
👏👏

File: package.json (hunk 2 above)
Comment (brandonroberts) [reply package.json:193]: These were added with the Nx upgrade as far as I know, but we shouldn't need them

File: modules/component/schematics-core/tsconfig.lib.json (hunk 1 above)
Comment (brandonroberts) [reply modules/component/schematics-core/tsconfig.lib.json:8]: Schematics do score, don't they 😄

**Pull request merged**